poptorch_experimental_addons/collectives.py

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) 2023 Graphcore Ltd. All rights reserved.
+
+from ._impl.collectives import *  # NOQA:F401,F403
+from ._impl.collectives import __all__, __doc__  # NOQA:F401
```

Comment:
Missing trailing newline (do you have autoformat on in the IDE?)

Reply:
I do, but I must be missing this for some reason